src/components/BlogCard/index.js

```diff
@@ -79,15 +138,79 @@ BlogCard.propTypes = {
       title: PropTypes.string.isRequired,
       author: PropTypes.string,
       date: PropTypes.string,
-      description: PropTypes.string,
-      version: PropTypes.string,
-    }).isRequired,
-    excerpt: PropTypes.string.isRequired,
+      path: PropTypes.string,
+    }),
+    title: PropTypes.string,
+    author: PropTypes.string,
+    date: PropTypes.string,
+    path: PropTypes.string,
+    excerpt: PropTypes.string,
     fields: PropTypes.shape({
       slug: PropTypes.string.isRequired,
       sourceInstanceName: PropTypes.string.isRequired,
     }),
   }).isRequired,
 };
 
-export default BlogCard;
+export const FeaturedBlogCard = ({ node, ...rest }) => (
+  <GrommetCard
+    elevation="medium"
+    flex="grow"
+    {...rest}
+    onClick={
+      node.frontmatter.path ? () => navigate(node.frontmatter.path) : undefined
+    }
+  >
+    <CardHeader justify="end" pad={{ vertical: 'small', horizontal: 'medium' }}>
+      <Text color="text-weak">{node.frontmatter.tags}</Text>
+    </CardHeader>
+    <Box align="start" pad="large" gap="medium">
+      <Box gap="small">
+        <Heading level={4} margin="none" color="text">
+          {node.frontmatter.title}
+        </Heading>
+        <Paragraph margin="none">
+          {node.frontmatter.description || stripMarkdown(node.excerpt)}
+        </Paragraph>
+      </Box>
+      <Box align="start" direction="row" gap="small">
+        <Image fit="contain" src="/img/blogs/Avatar1.svg" />
```

Comment:
What do you plan to use for the avatar? Maybe we can access peoples 'gravitar'? We do that for various other apps. We certainly need something other than the same image for everything. I definitely miss the look of the blog-specific header image. That's more inviting than someone's avatar

Reply:
Do you know how to fetch the first image on the first line of the markdown file and use it as a thumbnail image for the post? if not, then we can add a thumbnail image to the frontmatter and display.